pom.xml

```diff
@@ -16,18 +16,19 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- Dependencies version (external) -->
-    <org.apache.commons.commons-compress.version>1.22</org.apache.commons.commons-compress.version>
+    <org.apache.commons.commons-compress.version>1.25.0</org.apache.commons.commons-compress.version>
     <org.apache.commons.commons-lang3.version>3.12.0</org.apache.commons.commons-lang3.version>
     <commons-io.commons-io.version>2.11.0</commons-io.commons-io.version>
     <org.apache.commons.commons-email.version>1.5</org.apache.commons.commons-email.version>
     <org.freemarker.freemarker.version>2.3.32</org.freemarker.freemarker.version>
-    <com.google.guava.guava.version>31.1-jre</com.google.guava.guava.version>
-    <log4j.log4j.version>1.2.17</log4j.log4j.version>
+    <com.google.guava.guava.version>33.0.0-jre</com.google.guava.guava.version>
+    <log4j.log4j.version>2.22.1</log4j.log4j.version>
```

Comment:
Here the log4j version is upgraded to the major version 2, and reload4j is a log4j 1 fork, so there is a problem.